src/bin/cli.ts

```diff
@@ -170,8 +176,10 @@ const run = async function () {
       return successfulMigration
     } catch (err) {
       console.log(chalk`🚨  {bold.red Migration unsuccessful}`)
-      console.log(chalk`{red ${err.message}}`)
-      console.log(err.errors.map((err) => chalk`{red ${err.message}}\n`))
+      console.log(chalk`{red ${err.message}}\n`)
+      err.errors.forEach((err) => console.log(chalk`{red ${err}}\n\n`))
+      await Promise.all(serverErrorsWritten)
+      console.log('Please check the errors log for more details')
```

Comment:
Could we also log the name of the file so I can copy paste and `tail` it directly